VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,58 @@
+
+Back to the comparison, the AsyncAPI specification in the asynchronous world is analogous to OpenAPI in the synchronous world. 
+
+Both initiatives share numerous similarities:
+
+- Contract-first philosophy
+- Implementation-agnostic
+- Similar syntax
+- A tooling ecosystem 
+- Extensible
+- They protect information consumption using different security schemes. (Example: user and password, API keys, certificates, OAuth, OIDC, etc.)
+
+## Differences between AsyncAPI and OpenAPI
+
+As we have seen, the two initiatives are significantly similar, but they also cover significantly different fields. REST APIs have a specific function that requires their methods and documentation approaches. REST APIs have, among others, a great solution in the form of OpenAPI, a set of standardized documentation methods for enabling tools and interactivity.
+ 
+It's essential to understand that while the two solutions have a lot in common, they are also wildly divergent. Both OpenAPI and AsyncAPI allow for more significant cooperation and collaboration between different documented APIs. Still, OpenAPI is a solution for REST, and AsyncAPI is a solution for message-based architectures. Each aims to serve a different domain.
+
+## AsyncAPI and OpenAPI comparison
+
+Microservices architecture makes something pretty cool: to split up huge globs of code into tiny, manageable pieces. That’s the way different teams can work on pieces simultaneously, sharing the fun!
+
+Making different combinations of microservices appears them as the most efficient way of creating something new. Making combinations as a puzzle. REST calls were the main method to stick those pieces together, acting as glue.
+
+Jesse Menning makes it very clear in this fantastic [article](https://www.asyncapi.com/blog/openapi-vs-asyncapi-burning-questions) that we highly recommend.
+
+That method worked pretty well in synchronous communication, but it has its limitations. People eventually realized that REST wasn’t always the best tool for laying the game. The search for a new kind of glue starts as a way of facilitating asynchronous communication.
+
+The internet of things acted as a real game-changer. Since it was introduced, everything seems to be connected to the Internet. And everything Is everything, including vacuum cleaners, heaters, ovens, cars... That is amazing but not that easy. Every gadget doesn’t always have solid internet connections. Asynchronous communication rises as a better way to connect them.
+
+Establishing asynchronous communication between microservices makes them more reliable, faster, easier to scale, and more agile to adopt.
+
+![Protocols supported](https://user-images.githubusercontent.com/77982319/165746023-ed78afbc-598b-4401-843e-6ac851d30303.jpg)
```

Comment:
aws is not a protocol, try to see if they have some logo for SNS SQS protocols
please remove JMS and add Kafka logo instead